spec/price_policies_controller_shared_examples.rb

```diff
@@ -10,7 +10,8 @@
     @facility_account = @authable.facility_accounts.create(FactoryGirl.attributes_for(:facility_account))
 
     # Delete the default price groups since they get in the way of testing
-    PriceGroup.all.each(&:delete)
+    UserPriceGroupMember.delete_all
```

Comment:
Evidently so--it looks like nil (unpersisted) price policies are getting created for the default price groups by `PricePolicyBuilder` (called from `PricePoliciesController#new`).

`PricePolicyBuilder` is very difficult to read and I suspect a lot of weird stuff is happening in there. Possibly we should look into refactoring it, but I think that's probably out of scope for this already-large-enough PR.